### DISCUSSION TITLE
Errors when copying example code

### DISCUSSION THREAD

**Discussion by ghost** [tttapa/Control-Surface#962] [Q&A]
Hi there, I've put the following code into my arduino leonardo to try and get a midi note that has an LED attached to it, which i've copied and modified from the examples:

`#include <Control_Surface.h> 

USBMIDI_Interface midi;

NoteButton button1 {2, {MIDI_Notes::C(4), CHANNEL_1}};

NoteLED led1 {13, {MIDI_Notes::C(4), CHANNEL_1}};

void setup() {Control_Surface.begin();}
 
void loop() {Control_Surface.loop();}`

However I get this error:
Compilation error: 'CS::MIDI_Notes::C' cannot be used as a function

and if I switch out the MIDI_Notes function to a hex value instead, I get this error:
Compilation error: 'NoteLED' does not name a type

Can anyone help and tell me what I'm missing? thanks

**Comment by tttapa** [comment]
Looks like you have an older version of Control Surface installed. Try installing the version that's shown in the top left of the documentation page you used (most likely `main`).

**Reply by ghost** [reply]
I'm using version 1.2.0 which is the latest release as per the main page

**Reply by tttapa** [reply]
If you used an example from the `main` documentation, you should install the `main` version of Control Surface, not `1.2.0`.   

If for whatever reason you have to use an older version, you can find the documentation for that specific release on https://tttapa.github.io/Control-Surface-doc/.
